tests/e2e/steps/steps_def.go

```diff
@@ -71,31 +72,35 @@ func GetTestResources(valuesFilePath string) ([]Resource, error) {
 		return nil, fmt.Errorf("failed to read unmarshal values file: %v", err)
 	}
 
-	recourses := []Resource{}
+	resources := []Resource{}
 	for _, scene := range scenarios {
-		b, err := ioutil.ReadFile(scene.Path)
-		if err != nil {
-			return nil, fmt.Errorf("failed to read testdata: %v", err)
-		}
-
-		customResource := csmv1.ContainerStorageModule{}
-		err = yaml.Unmarshal(b, &customResource)
-		if err != nil {
-			return nil, fmt.Errorf("failed to read unmarshal CSM custom resource: %v", err)
+		customResources := []csmv1.ContainerStorageModule{}
+		for _, path := range scene.Paths {
+			b, err := ioutil.ReadFile(path)
+			if err != nil {
+				return nil, fmt.Errorf("failed to read testdata: %v", err)
+			}
+			customResource := csmv1.ContainerStorageModule{}
+			err = yaml.Unmarshal(b, &customResource)
+			if err != nil {
+				return nil, fmt.Errorf("failed to read unmarshal CSM custom resource: %v", err)
+			}
+			customResources = append(customResources, customResource)
 		}
-
-		recourses = append(recourses, Resource{
+		resources = append(resources, Resource{
 			Scenario:       scene,
-			CustomResource: customResource,
+			CustomResource: customResources,
 		})
 	}
 
-	return recourses, nil
+	return resources, nil
 }
 
-func (step *Step) applyCustomResource(res Resource) error {
-	cr := res.CustomResource
-	crBuff, err := ioutil.ReadFile(res.Scenario.Path)
+// JJL uses Scenario
```

Comment:
JJL uses scenario? Can this comment be more descriptive?

Reply:
whoops dev comment wil delete